products/views.py

```diff
@@ -27,5 +27,18 @@ def get(self, request, product_id):
         }
 
         return JsonResponse({'results': results}, status=200)
-            
-
+    
+class CategoryImageView(View):
+    def get(self, request):
+        categories = Category.objects.all()
+        results = []
+        
+        results.append(
+            {
+                'id'   : category.id,
+                'name' : category.name,
+                'image': category.image
+            } for category in categories
+        )
```

Comment:
```suggestion
        results = [{
                'id'   : category.id,
                'name' : category.name,
                'image': category.image
            } for category Category.objects.all()]
```
list comprehension의 형태는 다음과 같습니다. 정민님이 수정한 거랑 어떤 차이가 있는지 코멘트로 남겨 주세요.

Reply:
경훈님 죄송합니다,,,,,, 제가 급하게 수정하느라 수정에 오류가 있었습니다,,
다음부터는 더 잘 확인하고 push 하겠습니다,,!

list comprehension 형태로 할 경우, 기존 리스트에 append하는게 아니라 리스트에 바로 입력하며 생성하기때문에, 빈 리스트가 필요하지 않습니다.